canonical-web-and-design/maas-ui
# fix(ui): fix machine config forms not using updated values on save

## Done

- Removed `resetOnSave` prop from machine config forms, which was causing the form to be reset to before the values were changed.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the config form for a machine
- Change the pool, zone and note and click "Save"
- Check that the values stay updated

## Fixes

Fixes #2649 
Fixes #2651
